src/tracing/utils.rs

```diff
@@ -126,7 +126,7 @@ pub(crate) fn maybe_revert_reason(output: &[u8]) -> Option<String> {
         }
         GenericRevertReason::RawString(err) => err,
     };
-    if reason.is_empty() {
+    if reason.is_empty() || reason.trim_matches('\0').is_empty() {
```

Comment:
can we document this here

trying to find the geth equivalent for this, and unsure where the equivalent check for this is

maybe this here?

https://github.com/ethereum/go-ethereum/blob/85077be58edea572f29c3b1a6a055077f1a56a8b/accounts/abi/abi.go#L278-L281